Source/Core/Core/PowerPC/SignatureDB/SignatureDB.h

```diff
@@ -18,6 +18,51 @@ class SignatureDBFormatHandler;
 class SignatureDB
 {
 public:
+  enum class Handler
+  {
+    DSY,
+    CSV
+  };
+  SignatureDB(Handler handler = Handler::DSY);
```

Comment:
So why was being explicit about the database type being used not chosen across the board? Giving one format a 'lazy' escape hatch over the others seems kind of lopsided.